app/components/CheckmarkCircleLight.js

```diff
@@ -0,0 +1,26 @@
+const React = require('react');
+const { defineSvg } = require('../defineSvg');
+
+module.exports = defineSvg('CheckmarkCircleLight', {
+  className: 'tilt-icon-cherckmark-circle-light',
+
+  render() {
+    return (
+      <svg {...this.props} className={this.getClassNames() } viewBox="0 0 144 144">
+        <title>CheckmarkCircleLight</title>
+        <g className="tilt-icon-main">
+          <g transform="translate(-303.000000, -191.000000)" fill="#FFFFFF">
```

Comment:
If you remove the transforms, you could move the fill to the css under `tilt-icon-main`